docs/rst/tutorials/core/deployment/ds_wan_tcp/ds_wan_tcp.rst

```diff
@@ -256,7 +256,8 @@ Run the example:
 .. note::
 
     The requirements to achieve TCP communication over WAN with Discovery Server as EDP in a real deployment are simply launching the three elements of the communication (``talker``, ``listener`` and discovery *server*) with their corresponding XML configurations applied.
-    The remaining ``iptables`` and docker configuration and deployment have been defined to simulate WAN scenario locally.
+    To do so in a real scenario, setting the proper firewall or router configuration rules should be considered.
```

Comment:
Maybe we could add here a reference to the section [Configure transversal NAT on the network router
](https://eprosima-dds-router.readthedocs.io/en/latest/rst/use_cases/wan_tcp.html) of the *WAN over TCP DDS Router tutorial* to see an example on how to proceed